nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -476,7 +495,8 @@ def start_server_app(self, args):
 
         args.server_config = os.path.join("config", JobConstants.SERVER_JOB_CONFIG)
         app_custom_folder = os.path.join(app_server_root, "custom")
-        sys.path.append(app_custom_folder)
+        if os.path.isdir(app_custom_folder) and app_custom_folder not in sys.path:
+            sys.path.append(app_custom_folder)
```

Comment:
Why both simulator runner and simulator worker needs to add this custom path?
Is it because one represents the server side and the other is the client side?

Reply:
simulator runner and simulator worker are running in separate processes. They need to do the initialize separately.